src/app/questionForMode.js

```diff
@@ -0,0 +1,11 @@
+export const questionForMode = (mode) => {
+  const questions = {
```

Comment:
Perhaps you could add some validation? What if the mode comes in as undefined or any other value? What should happen then?